patternfly/patternfly-elements
# upgrading generator-pfelement to latest version

## upgrading generator-pfelement to latest version

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.
- upgrading package.json to version 1.3.0 of generator-pfelement
- adding `--type=pfelement` to `npm run new` so the generator knows that we want to create a new PatternFly Element inside the patternfly-elements repository

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. From the root of the patternfly-elements repository run `npm run new`
2. Create a new element (be sure to prefix it with "pfe-")
3. Validate: 
    - there is a a .story.js file in the demo directory of new element
    - in the gulpfile.js file, make sure the `gulpFactory` call receives `pfelementPackage` as the parameter
    - the package.json file should have a keywords array, repository.url should point to "github:patternfly/patternfly-elements" and repository.directory should point to "elements/whatever-component-you-created"
